README.md

```diff
@@ -0,0 +1,47 @@
+o-normalise [![Circle CI](https://circleci.com/gh/Financial-Times/o-normalise/tree/master.svg?style=svg)](https://circleci.com/gh/Financial-Times/o-normalise/tree/master)
+=================
+
+Foundation styles and standardised utilities
+
+- [Usage](#usage)
+	- [Sass](#sass)
+- [Contributing](#contributing)
+- [Contact](#contact)
+- [Licence](#licence)
+
+
+## Usage
+
+### Sass
+
+As with all Origami components, o-normalise has a [silent mode](http://origami.ft.com/docs/syntax/scss/#silent-styles). To use its compiled CSS (rather than using its mixins with your own Sass) set `$o-normalise-is-silent : false;` in your Sass before you import the o-normalise Sass.
+
+#### Available mixins
+
```

Comment:
I think it would be good to include a sentence as to why each of these are good? For example, 
- Clearfix one could just be a link to a discussion about clearing floats
- NormaliseBoxStyling works around a class of layout bug as described [here](some link). 
- etc